python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+    populated.
+
+    Parameters
+    ----------
+    datasetType : `str`
+        Dataset to lookup.
+    registry : `lsst.daf.butler.Registry`
+        Butler registry to query.
+    quantumDataId : `lsst.daf.butler.ExpandedDataCoordinate`
+        Data id to transform to identify crosstalkSources.  The
+        ``detector`` entry will be stripped.
+    collections : `lsst.daf.butler.CollectionSearch`
+        Collections to search through.
+
+    Returns
+    -------
+    results : `list` [`lsst.daf.butler.DatasetRef`]
+        List of datasets that match the query that will be used as
+        crosstalkSources.
+    """
+    newDataId = quantumDataId.subset(DimensionGraph(registry.dimensions, names=["instrument", "exposure"]))
+    results = set(registry.queryDatasets(datasetType, collections=collections, dataId=newDataId,
+                                         findFirst=True))
+    # In some contexts, calling `.expanded()` to expand all data IDs in the
+    # query results can be a lot faster because it vectorizes lookups.  But in
+    # this case, expandDataId shouldn't need to hit the database at all in the
+    # steady state, because only the detector record is unknown and those are
+    # cached in the registry.
+    return [ref.expanded(registry.expandDataId(ref.dataId, records=newDataId.records)) for ref in results]
+
+
+class IsrTaskLSSTConnections(pipeBase.PipelineTaskConnections,
+                             dimensions={"instrument", "exposure", "detector"},
+                             defaultTemplates={}):
+    ccdExposure = cT.Input(
+        name="raw",
+        doc="Input exposure to process.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    camera = cT.PrerequisiteInput(
+        name="camera",
+        storageClass="Camera",
+        doc="Input camera to construct complete exposures.",
+        dimensions=["instrument"],
+        isCalibration=True,
+    )
+    dnlLUT = cT.PrerequisiteInput(
+        name="dnlLUT",
+        doc="Look-up table for differential non-linearity.",
+        storageClass="IsrCalib",
+        dimensions=["instrument", "exposure", "detector"],
+        isCalibration=True,
+        # TODO DM 36636
+    )
+    bias = cT.PrerequisiteInput(
+        name="bias",
+        doc="Input bias calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    deferredChargeCalib = cT.PrerequisiteInput(
+        name="cpCtiCalib",
+        doc="Deferred charge/CTI correction dataset.",
+        storageClass="IsrCalib",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    linearizer = cT.PrerequisiteInput(
+        name='linearizer',
+        storageClass="Linearizer",
+        doc="Linearity correction calibration.",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+        minimum=0,  # can fall back to cameraGeom
```

Comment:
I am not sure what this means; we should either use a linearizer calibration or not.  I think that automated fallbacks can be dangerous and confusing.

Reply:
this is copied from the original task so I won't address it in this ticket.

Reply:
I think you can just remove `minimum=0`.